#input  
int64 objectId
bool use_planner
---
#output
float64 heightOfObject
bool success
---
#feedback
int64 curState
string curStateInfo